### ISSUE TITLE
Issue to trigger new releases

### ISSUE THREAD

**Issue by ablaom** [FluxML/MLJFlux.jl#47]
@JuliaRegistrator register()

**Comment by ablaom** [comment]
@JuliaRegistrator register()

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
JuliaRegistrator() register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@DilumAluthge I've already updated the url: https://github.com/JuliaRegistries/General/pull/30676

Any idea why this isn't working.

Same result on JuliaHub

**Comment by DilumAluthge** [comment]
@JuliaRegistrator register

**Comment by DilumAluthge** [comment]
https://github.com/JuliaRegistries/General/pull/30702

**Comment by ablaom** [comment]
#96

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegister register

release notes:

- [x] Add support for MLJ's forthcoming iteration interface (#100, https://github.com/alan-turing-institute/MLJ.jl/issues/139)

**Comment by ablaom** [comment]
@JuliaRegistrator register

release notes:

- [x] extend compat for Flux to include 0.12
- [x] extend compat for MLJModelInterface to include 1.0

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

release notes:

- [x] Remove type piracy for optimisers (#135)
- [x] Fix incorrect cold-start behaviour (#124)
- [x] Make `builder` and `optimiser` "deep" hyper-parameters (#135)

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

release notes:

- [ ] (**enhancement**) Add `rng` hyper-parameter to all models, for passing to builders (#166)
- [ ] (**breaking**) Extend the signature of `build` to include an RNG, as in `build(model, rng, shape...)` (#166)

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

- [x] (**enhancement**) Add convince macro `@builder` for defining builders (#177) @jbrea

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

- [x] (**bug fix**) Enable L2/L1 regularisation parameters `alpha` and `lambda` to work as intended (#179)

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

release notes:

- [x] Add builder for multilayer perceptron, `MLJFlux.MLP` (#188 #190) @john-waczak

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

Release notes:

- #212

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

Release notes:

- (**bug fix**) Address improper scaling of l2/l1 loss penalty with batch size (#213) @mohamed82008

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

Release notes:

- (**mildy breaking**) Extend compatibility requirement: Flux = "0.13, 0.14". If using Flux 0.14 or later, it may be necessary to additionally import `cuDNN` to recover the functionality of previous versions (e.g. if using convolution neural networks in builders) (#232)

- Bump compatibility requirement: Metalhead = "0.8"

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

Release notes:

- Improve the error message for faulty builders (#238)

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

Release notes:

- (**new model**) Add `NeuralNetworkBinaryClasssifier`, an optimised form of `NeuralNetworkClassifier` for the special case of two target classes. Use `Flux.σ` instead of `softmax` for the default finaliser (#248)
- (**internals**) Switch from implicit to explicit differentiation (#251)
- (**breaking**) Use optimisers from Optimisers.jl instead of Flux.jl (#251). Note that the new optimisers are immutable.
- (**RNG changes.**) Change the default value of the model field `rng` from`Random.GLOBAL_RNG` to `Random.default_rng()`. Change the seeded RNG, obtained by specifying an integer value for `rng`, from `MersenneTwister` to `Xoshiro` (#251)
- (**RNG changes.**) Update the `Short` builder so that the `rng` argument of `build(::Short, rng, ...)`
   is passed on to the `Dropout` layer, as these layers now support this on a GPU, at
  least for `rng=Random.default_rng()` (#251)
- (**weakly breaking**) Change the implementation of L1/L2 regularization from explicit loss penalization to  weight/sign decay (internally chained with the user-specified optimiser). The only breakage for users is that the losses reported in the history will no longer be penalized, because the penalty is not explicitly computed (#251)

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

Release notes:

All models, except `ImageClassifier`, now support **categorical features** (presented as table columns with a `CategoricalVector` type). Rather than one-hot encoding, embeddings into a continuous space are learned (i.e, by adding an  embedding layer) and the dimension of theses spaces can be specified by the user, using a new dictionary-valued hyperparameter, `embedding_dims`.  See [Cheng Guo and Felix Berkhahn (2016)](https://arxiv.org/abs/1604.06737): Entity Embeddings of Categorical Variables.

Also, all continuous input presented to these models is now forced to be `Float32`, but this is the only breaking change.

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

Release notes:

- Add model wrapper `EntityEmbedder(model)` to transform supervised MLJFlux models into entity embedding transformers (#286)
- Make some performance improvements around unwrapping of `CategoricalArray`s (#281)

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

Release notes:

-  Extend compatibility: Flux = "0.14, 0.15"

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

Release notes:

- (**bug fix**) Resolve issue around optimisers not updating after warm restart (#322)
- (**deprecation**) Some signatures of the internal methods `train/train_epoch` have been deprecated (#324)
